### PR TITLE
Add option to reboot nodes sequentially

### PR DESCRIPTION
**Pull request by olegBuda**

### CODE REVIEW

File: roles/install_storpool_components/handlers/main.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@
   ansible.builtin.reboot:
     reboot_timeout: 900
   listen: Reboot host
+  throttle: "{{ 1 if sp_reboot_sequentially else 63 }}"
```
Comment (netcho) [inline roles/install_storpool_components/handlers/main.yml:32]: Better use the `ternary` filter

**Review by netcho — COMMENTED** [review]
Looks fine

File: roles/install_storpool_components/handlers/main.yml (hunk 1 above)
Comment (olegBuda) [reply roles/install_storpool_components/handlers/main.yml:32]: I thought this was python's ternary operator? If you mean something else - I need more explanations :)

Comment (netcho) [reply roles/install_storpool_components/handlers/main.yml:32]: I'm referring to this: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/ternary_filter.html

**Review by netcho — APPROVED** [review]

**Pull request merged**